src/componenets/modal/AddAtFolder.tsx

```diff
@@ -1,12 +1,22 @@
+import { SyntheticEvent } from "react";
 import close from "../../images/_close.png";
 
-function AddAtFolder({ toggleModal }) {
+type AddAtFolderType = {
+  toggleModal: (id: number) => void;
+};
+
+function AddAtFolder({ toggleModal }: AddAtFolderType) {
+  function handleImageClick(event: SyntheticEvent) {
```

Comment:
이런식으로 타입을 잡아주면 더 좋을 것 같아요~
```suggestion
  function handleImageClick(event: MouseEvent<HTMLImgElement>) {
```